### PR TITLE
force aria-live as ariaLve in Spinner not working

### PR DESCRIPTION
**Pull request by alcail**
# What
used the AriaAttributes aria-live instead of existing ariaLive prop from Spinner.

# Why
Issue is our spinner is embedded in a Stack and in that case the ariaLive prop of Spinner component is not working properly ending in Spinner label not being read by screen reader
I checked this FluentUI component ariaLive prop in its doc and it's indeed not working properly when embedded in a Stack.

# How Tested
ran Call sample locally with 2 participants, shared the screen from 1 of them and checked the screen reader announcement for the other one.

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Review by JamesBurnside — APPROVED** [review]

File: packages/react-components/src/components/VideoGallery/RemoteScreenShare.tsx [hunk 1]
```diff
@@ -60,7 +60,7 @@ export const RemoteScreenShare = React.memo(
 const LoadingSpinner = (props: { loadingMessage: string }): JSX.Element => {
   return (
     <Stack verticalAlign="center" className={loadingStyle}>
-      <Spinner label={props.loadingMessage} size={SpinnerSize.xSmall} />
+      <Spinner label={props.loadingMessage} size={SpinnerSize.xSmall} aria-live={'assertive'} />
```
Comment (alkwa-msft) [inline packages/react-components/src/components/VideoGallery/RemoteScreenShare.tsx:63]: is there a constant we can use here? I can imagine someone could just miss spell this

**Review by alkwa-msft — APPROVED** [review]

File: packages/react-components/src/components/VideoGallery/RemoteScreenShare.tsx (hunk 1 above)
Comment (alcail) [reply packages/react-components/src/components/VideoGallery/RemoteScreenShare.tsx:63]: we can create one, sure

Comment (JamesBurnside) [reply packages/react-components/src/components/VideoGallery/RemoteScreenShare.tsx:63]: I would actually argue against creating a constant here as aria-live has a specific interface: 
image

Comment (JamesBurnside) [reply packages/react-components/src/components/VideoGallery/RemoteScreenShare.tsx:63]: looking at that screenshot looks like we need to use ariaLive and not aria-live to get that type checking though

Comment (alcail) [reply packages/react-components/src/components/VideoGallery/RemoteScreenShare.tsx:63]: will do in another PR though

Comment (alcail) [reply packages/react-components/src/components/VideoGallery/RemoteScreenShare.tsx:63]: @JamesBurnside 'ariaLive' prop from Spinner FluientUI component is not working properly when inside a Stack, that's why I'm using the 'aria-live' from AriaAttributes to force the behaviour of Spinner

Comment (alcail) [reply packages/react-components/src/components/VideoGallery/RemoteScreenShare.tsx:63]: > 

the 'ariaLive' prop of Spinner component has indeed a specific interface, but not the 'aria-live' attribute

Comment (alcail) [reply packages/react-components/src/components/VideoGallery/RemoteScreenShare.tsx:63]: but, yes, you're right, even the AriaAttributes have specific interface, so no need for constants here then

**Pull request merged**